src/WMTLogger.ts

```diff
@@ -45,23 +48,43 @@ export class WMTLogger {
     /** Include time in the logs? */
     public static includeTime: boolean = true
 
-    static debug(message: string | any) {
+    /** Optional log listener. */
+    private static logListener: WMTLogListener | null = null
+
+    /** Whether the log listener should follow the current verbosity level. */
+    private static logListenerFollowsVerbosity: boolean = true
+
+    /** Sets the log listener called when a message is logged. */
+    static setLogListener(
+        listener: WMTLogListener | null,
+        followVerbosity: boolean = true
+    ) {
+        if (!listener) {
```

Comment:
I think the if is unnecessary - the result will be the same as if there were just 

```
this.logListener = listener
this.logListenerFollowsVerbosity = followVerbosity
```